hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/query/matcher/MatcherInputException.java

```diff
@@ -0,0 +1,7 @@
+package pl.allegro.tech.hermes.management.infrastructure.query.matcher;
+
+public class MatcherInputException extends RuntimeException {
```

Comment:
please extend `ManagementException`, so you'll be able to get rid of `MatcherExceptionMapper`